gramps/gen/db/generic.py

```diff
@@ -311,27 +325,15 @@ def __next__(self):
     def __exit__(self, *args, **kwargs):
         pass
 
-    def iter(self):
```

Comment:
I don't think that we should remove the `iter`, `first` and `next` methods.  They form part of the public API and third-party code may be using them.  If we want to remove these in the future then they should be deprecated first.

Reply:
Restored them.